sdk/storage/storage-file-share/samples/javascript/README.md

```diff
@@ -0,0 +1,68 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+- azure-storage
+urlFragment: storage-file-share-javascript
```

Comment:
@kaerm my understanding is that this `urlFragment` directive here will make it so the sample URL ends up being pinned to docs.microsoft.com/samples/Azure/azure-sdk-for-js/storage-file-share-javascript , right? Is there a preferred naming scheme for this?

Reply:
that's how I understand it as well

Reply:
Is there any preview link for the generted docs?

Reply:
i do not think so. Once we have it merged we can see how it generates, and if change is needed it will be automatically picked up